requirements.txt

```diff
@@ -1,11 +1,11 @@
-appdirs==1.4.3
-coverage==4.4
-packaging==16.8
-pbr==3.0.0
-py==1.4.33
-pyparsing==2.2.0
-pytest==3.0.7
+barbequeue==0.0.1
+enum34==1.1.6
+funcsigs==1.0.2
+futures==3.1.1
+mock==2.0.0
+pbr==3.0.1
+py==1.4.34
+pytest==3.1.1
 pytest-mock==1.6.0
-pytest-cov
```

Comment:
This is the problem you're looking at in current test failures in #13 @aronasorman 

`coverage` is also missing

Could also perhaps look a bit at how much dependencies should be pinned on minor version level since this is passed into the setup.py file.